ui/src/app/cohort-review/annotation-item/annotation-item.component.css

```diff
@@ -29,3 +29,113 @@ input[type=text]:not([readonly]):focus {
   border-right: none;
   border-left: none;
 }
+
+.textarea-wrapper {
+  border-radius: 7px;
+  background: #FFFFFF;
+  padding-top: 0.4rem;
+  padding-left: 0.6rem;
+  padding-right: 1.5rem;
+  height:7vh;
+  width:15.3rem;
+}
+
+.items-text {
+  color:#224F83;
+  font-weight: bold;
+  padding-bottom: 0.15rem;
+  padding-top: 0.6rem;
+}
+
+.participant-dropdown.dropdown, .dropdown .dropdown-toggle {
+  width:10rem;
+  margin-top: 0.2rem;
+  text-align: left;
+  margin-bottom: 0.4rem;
+}
+
+.btn-info-outline .btn, .btn-outline .btn, .btn.btn-info-outline, .btn.btn-outline, .btn.btn-outline-info, .btn.btn-outline .btn {
+  border : 1px solid #C3C3C3;
+  color: #4A4A4A;
+  background: #FFFFFF;
+}
+
+.dropdown .dropdown-toggle.btn clr-icon[shape^=caret] {
+  color: #C3C3C3;
+}
+
+.dropdown-menu {
+  margin-top: -0.35rem;
+  width:10rem;
+}
+
+.checkbox-container {
+  position: relative;
+  margin-top: 0.9rem;
+  margin-bottom:0.9rem;
+}
+
+.checkbox-container  .checkbox-style{
+  width: 1rem;
+  height: 1rem;
+  position: absolute;
+  left: 7rem;
+  top: 0rem;
+}
+
+.integer-style {
+  width: 6rem;
+  border:  1px solid #C3C3C3;
+  border-radius: 0.2rem;
+  background-color: #FFFFFF!important;
+  text-align: right;
+  padding-top: 0.7rem;
+  padding-bottom: 0.7rem;
+}
+
+.items-date {
+  color:#224F83;
+  font-weight: bold;
+  padding-bottom: 0.15rem;
+}
+
+.items-Integer {
```

Comment:
Nit: Is there any reason the "I" is capitalized here? If not, we should change to lowercase to be consistent